plugins/coatils.py

```diff
@@ -163,7 +165,9 @@ def position(stl, stc, enl, enc):
                                                               enl, enc)
 
     # Ignore PycodestyleBear, LineLengthBear
-    @re_botcmd(pattern=r'^run\s+(\w+)((?:\s+\w+(?:\s+\w+=\w+)*)+)\n+```\n([\s\S]+)\n```$')
+    @re_botcmd(pattern=r'^run\s+(\w+)((?:\s+\w+(?:\s+\w+=\w+)*)+)\n+```\n([\s\S]+)\n```$',
+               re_cmd_name_help='run <Bear [[setting=value]+]>+\n'
+                                '```\n<code>+\n```')
```

Comment:
hmm.. how else do I convey that "bear name followed by settings" should occur at least once(hence `<>`), and providing settings is optional, hence `[]`

Same for the code, hence `<code>`